housekeeping/docker/release/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && apt-get upgrade -y && \
+    apt-get install --no-install-recommends -y \
+        gcc-8 \
```

Comment:
gcc-8 installed to fix error.
```
error while loading shared libraries: libasan.so.5: cannot open shared object file: No such file or directory
```

Reply:
also I had trouble to run binary build on Ubuntu in alpine docker.

Reply:
I think to install accorded libasan would been enough

Reply:
What's troubles your got in alpine linux? Usually single troble is getting appropriate alpine packages =)